config/initializers/rack_attack.rb

```diff
@@ -57,15 +63,31 @@ def headers
         limit: IdentityConfig.store.requests_per_ip_limit,
         period: IdentityConfig.store.requests_per_ip_period,
       ) do |req|
-        req.remote_ip unless req.path.starts_with?('/assets') || req.path.starts_with?('/packs')
+        next if req.path.starts_with?('/assets') || req.path.starts_with?('/packs')
+        next if IdentityConfig.store.requests_per_ip_path_prefixes_allowlist.any? do |x|
+          req.path.starts_with?(x)
+        end
+        next if ALLOWED_CIDR_BLOCKS.any? do |cidr_block|
+          cidr_block === req.remote_ip
```

Comment:
I know `===` is an alias on this class but personally I find the word easier to read:
```suggestion
          cidr_block.include?(req.remote_ip)
```